backend/lib/utils.js

```diff
@@ -17,8 +19,32 @@ const bool = (env) => env == "true";
 
 const getCookieToken = (req) => req.cookies.token;
 
-const emailRegEx = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
-const isValidEmail = (email) => emailRegEx.test(email);
+// email validation code from front-end 'src\utils\validators'
+const isValidTopLevelDomain = (string) => {
+  return tlds.some(tld => string.endsWith("." + tld));
+};
+
+const isValidEmail = (email) => {
+  return _isEmail(email) && isValidTopLevelDomain(email);
+};
+
+// password validation code from front-end 'src\utils\validators'
+const SPECIAL_CHARS = /[!@#$%^&*.;]/;
+const containsLowerCase = (str) => /[a-z]/.test(str);
+const containsUpperCase = (str) => /[A-Z]/.test(str);
+const containsNumber = (str) => /\d/.test(str);
+const containsSpecialChars = (str) => SPECIAL_CHARS.test(str);
+
+const isValidPassword = (password) => {
+  const checks = [
+    containsLowerCase(password),
+    containsUpperCase(password),
+    containsNumber(password),
+    containsSpecialChars(password),
+  ];
+
+  return checks.filter((check) => check).length >= 3;
```

Comment:
```suggestion
  const passwordChecks = [
    containsLowerCase(password),
    containsUpperCase(password),
    containsNumber(password),
    containsSpecialChars(password),
  ];

  return passwordChecks.filter(isValid => isValid).length >= 3;
```

Reply:
Thanks JV, that's better, I commit it : )